proto/frequenz/api/microgrid/v1/microgrid.proto

```diff
@@ -413,15 +413,25 @@ message ListConnectionsResponse {
 
 // Request parameters for the RPC `ReceiveComponentDataStream`.
 message ReceiveComponentDataStreamRequest {
+  // A message for specifying a filter to apply to the stream.
+  message ComponentDataStreamFilter {
+    // List of metrics to return. Only the specified metrics will be returned.
+    // A complete list of metrics can be found [here](https://github.com/frequenz-floss/frequenz-api-common/blob/v0.5.0/proto/frequenz/api/common/v1/metrics/metric_sample.proto#L65-L66).
+    //
+    // !!! note
+    //     At least one metric must be specified. Failure to do so will result in an empty response.
```

Comment:
In the commit is says the RPC will return an error. Maybe also add this here in the docs.

Reply:
Good catch, thanks! Updated.